### PR TITLE
`CONTRIBUTING.md` basics section

### PR DESCRIPTION
**Pull request by aphelionz**
## Motivation

`snarkVM` is central to the entire Aleo ecosystem. The effects of bugs or even small changes to this codebase ripple outward from here and have unintended consequences, so it's imperative that software engineering best practices are followed to match the mathematical and cryptographic precision required by the VM.

The changes proposed are a mild set of quick and easily-enforceable idioms that EQ has found to improve not only the quality of the code, but the quality of experience of _working with_ the code.

Note: Here's the ink to [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) to draw it out of the `.md` file in the changeset

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,6 +83,11 @@ Now to use `snarkvm`, in your terminal, run:
 snarkvm
 ```
 
+## 3. Contributing
+
+PRs and issues are welcome. Please see the [contribution guide](./CONTRIBUTING.md) for more
+information about how to create a successful PR.
+
 ## 3. Usage Guide
```
Comment (howardwu) [inline README.md:91]: Nit: the numbering and ToC needs to be updated

Comment (aphelionz) [reply README.md:91]: It looks like GitHub has recently added a feature to auto-generate README ToCs. Should we try that?

Comment (aphelionz) [reply README.md:91]: And/or perhaps we drop the `3. Usage Guide` stub for now?

Comment (aphelionz) [reply README.md]: For now just updated it without the automated GitHub ToC

File: README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@
 
 * [1. Overview](#1-overview)
 * [2. Build Guide](#2-build-guide)
-* [3. Usage Guide](#3-usage-guide)
+* [3. Usage Guide](#3-contributing)
```
Comment (howardwu) [inline README.md:15]:
```suggestion
* [3. Contributing](#3-contributing)
```

Comment (aphelionz) [reply README.md]: Fixed with `git commit --amend` :+1:

**Review by howardwu — APPROVED** [review]

**Pull request merged**